src/Core/SharedKernels/Questionnaire/WB.Core.SharedKernels.Questionnaire/Documents/Question/TextQuestion.cs

```diff
@@ -20,7 +22,7 @@ public override void AddAnswer(Answer answer)
 
         public override IEnumerable<T> Find<T>(Func<T, bool> condition)
         {

```

Comment:
need remove this method and need inhered this class from 'ExternalServiceQuestion'